static/js/components/forms/B2BPurchaseForm_test.js

```diff
@@ -125,6 +123,26 @@ describe("B2BPurchaseForm", () => {
       )
     })
 
+    it("should validate email field", async () => {
+      const values = { email: "" }
+      try {
+        await emailValidation.validate(values)
+        assert.fail("Email validation should have thrown an error")
+      } catch (error) {
+        assert.strictEqual(error.message, "Email is a required field")
+      }
+    })
+
+    it("should validate email field is invalid", async () => {
+      const values = { email: "invalid-email" }
+      try {
+        await emailValidation.validate(values)
+        assert.fail("Email validation should have thrown an error")
+      } catch (error) {
+        assert.strictEqual(error.message, "Invalid email")
+      }
+    })
```

Comment:
I think this can be written in a single test with multiple input values. The values could be `"", "something", "something@", "@something"` so that all the cases are tested properly.

Reply:
fixed